### PR TITLE
Add Subscribers.Invoke

### PR DESCRIPTION
**Pull request by ddddxxx**
Passing published values to consuming method is very common:

```swift
somePublisher.sink { [weak self] value in
    self?.handleEvent(value)
}
```

It can be simplified like this:

```swift
somePublisher.invoke(handleEvent, on: self)
```

### CODE REVIEW

File: Sources/CXExtensions/Invoke.swift [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+    }
+}
+
+extension Subscribers.Invoke {
+    
+    public convenience init(object: Target, method: @escaping (Target) -> (Input) -> Void) {
+        self.init(object: .strong(object), method: .withParameter(method))
+    }
+    
+    public convenience init(objectNoRetain object: Target, method: @escaping (Target) -> (Input) -> Void) {
```
Comment (ddddxxx) [inline Sources/CXExtensions/Invoke.swift]: `Invoke.init(objectNoRetain:method:)` or `Invoke.init(weakObject:method:)`?

Comment (luoxiu) [reply Sources/CXExtensions/Invoke.swift]: `NSValue.init(nonretainedObject anObject: Any?)` follow 这个？

Comment (ddddxxx) [reply Sources/CXExtensions/Invoke.swift]: 这样会依赖 Foundation，而且 linux 下似乎不能用。

Comment (ddddxxx) [reply Sources/CXExtensions/Invoke.swift]: Done

**Review by luoxiu — APPROVED** [review]

**Pull request merged**